managed/services/checks/checks.go

```diff
@@ -1222,16 +1249,59 @@ func convertVMValue(value model.Value) ([]byte, error) {
 		return nil, errors.WithStack(err)
 	}
 
-	var data []map[string]interface{}
+	var data []map[string]any
 	if err = json.Unmarshal(b, &data); err != nil {
 		return nil, errors.WithStack(err)
 	}
 
-	res, err := agentpb.MarshalActionQueryDocsResult(data)
+	return data, nil
+}
+
+func (s *Service) discoverAvailablePGDatabases(ctx context.Context, target services.Target) ([]string, error) {
+	query := check.Query{Query: `datname FROM pg_database  
+WHERE datallowconn = true AND datistemplate = false AND has_database_privilege(current_user, datname, 'connect')`}
```

Comment:
Can we:
1. Add a link with description of parameters of the query;
2. Move query string to the `const` section?